core/examples/luigi-sample-angular/src/app/project/project.component.ts

```diff
@@ -12,7 +12,7 @@ import {
   uxManager,
   addContextUpdateListener,
   removeContextUpdateListener
-} from '@kyma-project/luigi-client/luigi-client-legacy';
+} from '@kyma-project/luigi-client';
```

Comment:
we don't support ie11 in our example, right?

Reply:
Correct, this it out of scope for this task, I will add a comment in the readme